.storybook/webpack.config.js

```diff
@@ -1,30 +1,41 @@
 const TsconfigPathsPlugin = require("tsconfig-paths-webpack-plugin");
 const HardSourceWebpackPlugin = require("hard-source-webpack-plugin");
 const WebpackBar = require("webpackbar");
-const ProgressPlugin = require("webpack").ProgressPlugin;
+const webpack = require("webpack");
+
+const LANGUAGES_REGEX = require("../config/languagesRegex");
 
 module.exports = ({ config, mode }) => {
   config.module.rules = config.module.rules.filter(
     (r) => r.test.toString() !== /\.md$/.toString(),
   );
   config.plugins = config.plugins.filter(
-    (m) => m instanceof ProgressPlugin === false,
+    (m) => m instanceof webpack.ProgressPlugin === false,
   ); // We are using Webpackbar, so no need in ProgressPlugin
   config.plugins.push(new HardSourceWebpackPlugin());
   config.plugins.push(new WebpackBar());
+  config.plugins.push(
+    new webpack.ContextReplacementPlugin(/moment[/\\]locale$/, LANGUAGES_REGEX),
```

Comment:
@goooseman I can remove this for moment, but for date-fns it cannot be removed because webpack includes everything that is in the require context (including md files, flow etc.)

Reply:
Ok